cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/AuthorizationHandlerTest.java

```diff
@@ -416,6 +397,33 @@ public void testRevokeAllOnNonExistingEntity()
     client.revoke(Ids.namespace("ns3"));
   }
 
+  @Test
+  public void testWildCardEntities() throws Exception {
+    // tin-mem authorizer does not support wildcard privileges so we can't actually enforce after grant.
+    // this test is just to check that our grant apis work fine. After grant we list privileges to check for existence
+    StreamId streamId = new StreamId("ns", "wildcard");
+    String wildcardEntityStar = streamId.toString().replace("wildcard", "*");
+    client.grant(Authorizable.fromString(wildcardEntityStar), admin, ImmutableSet.of(Action.ADMIN));
+    String wildcardEntityQuestion = streamId.toString().replace("wildcard", "someSt?");
+    client.grant(Authorizable.fromString(wildcardEntityQuestion), admin, ImmutableSet.of(Action.ADMIN));
+
+    Set<Privilege> privileges = client.listPrivileges(admin);
```

Comment:
Just to confirm - as a user, I can directly give the authorizable from the output of the `listPrivileges` method to the `revoke` method, right? Just want to make sure that output of the `listPrivileges` is in the same format as the input of `revoke` or `grant` method.